cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/internal/BootstrapCoordinator.scala

```diff
@@ -192,15 +235,20 @@ private[akka] final class BootstrapCoordinator(discovery: SimpleServiceDiscovery
     case ProbingFailed(contactPoint, _) =>
       lastContactsObservation.foreach { contacts =>
         if (contacts.observedContactPoints.contains(contactPoint)) {
-          log.info("Received signal that probing has failed, scheduling contact point re-discovery")
+          log.info("Received signal that probing has failed, scheduling contact point probing again")
+          // child actor will have terminated now, so we ride on another discovery round to cause looking up
+          // target nodes and if the same still exists, that would cause probing it again
+          //
+          // we do this in order to not keep probing nodes which simply have been removed from the deployment
         }
       }
 
       // remove the previous observation since it might be obsolete
       seedNodesObservations -= contactPoint
+      startSingleDiscoveryTimer()
```

Comment:
not needed, but I guess it doesn't hurt to wait another interval before discovery here

Reply:
Right, here it really it just slows it down by rescheduling the same timer key again -- I'll add a comment